static/templates/settings/organization-profile-admin.handlebars

```diff
@@ -66,8 +67,7 @@
             </div>
         </div>
 
-        <h3>{{t "Organization logo for night mode" }}</h3>
-        <p>{{t "Like Organization logo, but for the night theme." }}</p>
+        <p>{{t "Logo for dark background" }}</p>
```

Comment:
What if we just call this "Alternative logo for dark background" or something?

Reply:
yeah, I waffled between that and what I chose. I think "Alternative logo .." is reasonable.